packages/docs/src/styles/_examples.scss

```diff
@@ -65,7 +65,7 @@ $options-margin: $pt-grid-size * 3;
     margin-bottom: $options-margin;
   }
 
-  :not(.pt-control-group) .pt-input-group + .pt-input-group {
+  :not(.pt-control-group) > .pt-input-group + .pt-input-group {
```

Comment:
This broken style was reappearing because the new `Popover` parent was now satisfying the `:not(.pt-control-group)` part of this selector. Requiring that this style apply only to `pt-input-group`s that are immediate children of non-`pt-control-group`s fixes the problem.